ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -1385,3 +1385,68 @@ def test_get_logical_disks(self, mock_list_virtual_disks):
 
         self.assertEqual({'logical_disks': [expected_logical_disk]},
                          props)
+
+    @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
+                       autospec=True)
+    def test_reset_idrac_configuration(self, mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            return_value = task.driver.raid.reset_idrac(task)
+
+            mock_client.reset_idrac.assert_called_once_with(
+                force=True, wait=True)
+
+        self.assertIsNone(return_value)
+        self.node.refresh()
+
+    @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
```

Comment:
It looks to me like this test is the same as the one above.  If so, can you remove one of them?

Reply:
I have removed this test case, as we don't makes any changes in raid.py.

Reply:
Can you back out the changes to this file?  This file is still showing up as modified.

Reply:
Done